src/stream/src/executor/dispatch.rs

```diff
@@ -152,6 +178,7 @@ struct DispatchExecutorInner {
     actor_id_str: String,
     context: Arc<SharedContext>,
     metrics: Arc<StreamingMetrics>,
+    actor_output_buffer_monitor_tasks: HashMap<ActorId, JoinHandle<()>>,
```

Comment:
Every dispatch executor corresponds to one actor, why do we need to store a HashMap (which might corresponds to multiple actors) here?